### ISSUE TITLE
Target user is not specific in the user guide

### ISSUE THREAD

**Issue by Judy1x4** [Judy1x4/pe#9]
![image.png](https://raw.githubusercontent.com/Judy1x4/pe/main/files/f7cbfb0a-4f68-4181-9b9f-f1301e26a66e.png)

The target user is not clear enough. The value proposition is not fully elaborated. 

The validation of fee field being compulsory is not explained fully nor adheres to the value proposition provided.
 
![image.png](https://raw.githubusercontent.com/Judy1x4/pe/main/files/a42be379-47d7-4fa5-80d4-ce23da3c215e.png)

**Comment by nus-pe-script** [comment]
[IMPORTANT!: Please do not edit or reply to this comment using the GitHub UI. You can respond to it using CATcher during the next phase of the PE]

# Team's Response
There seems to be 3 bugs suggested by this tester - that the target user is not clear enough, that the value proposition is not fully elaborated, and that the fee validation could have been further fleshed out.

We will consider the first bug which may seem more severe. As also written in #1512, we agree that "educators" could have been more specific (we were intending for "undergrad tutors"), but the severity should be lowered to `Medium` as specifying educators is enough, in our opinion, to make our features for student and lesson management valuable.

## The 'Original' Bug

[The team marked this bug as a duplicate of the following bug]

># Target user not well defined in UG
>
>**Description**
>The user guide just say "EduLog User Guide for Educators", this initially gave me the impression that this is an app for teachers and educators teaching in school. But then when I continued on to read the features about fee, mark, and unmark -> these features didn't make sense to me. As generally such administrative tasks are done by the school for the educators. Only after reading the developer guide (meant for developers), I am beginning to think that EduLog is made for those private tutors, who give personalized tutions to students. This should be made clear and very apparent in the user guide, so EduLog doesn't attract the wrong users. As this EduLog doesn't provide much value to educators linked to established institutions with a separate department for administrative tasks.
>
>**Screenshot**
>
>![Screenshot 2024-11-15 at 4.30.02 PM.png](https://raw.githubusercontent.com/nus-cs2103-AY2425S1/pe/master/files/3b15d2b1-1e4b-4f7d-a13b-c7315e02a4dc.png)
>
>
>
>
>
>
>
><hr>
><sub>[original: nus-cs2103-AY2425S1/pe-interim#2087] [original labels: severity.Medium type.DocumentationBug]</sub>
>
>
>

## Their Response to the 'Original' Bug

[This is the team's response to the above 'original' bug]

>You are right, good catch! Thank you for your efforts, we will fix our product accordingly.

# Items for the Tester to Verify

## :question: Issue duplicate status

Team chose to mark this issue as a duplicate of another issue (as explained in the _**Team's response**_ above)

- [ ] I disagree

**Reason for disagreement:** [replace this with your reason]

<catcher-end-of-segment><hr>

## :question: Issue severity

Team chose [`severity.Medium`]
Originally [`severity.High`]

- [ ] I disagree

**Reason for disagreement:** [replace this with your reason]

<catcher-end-of-segment><hr>
